docroot/status/index.php

```diff
@@ -212,24 +215,48 @@ function getStatusIcon($level) {
                 "SECCHI" => $providers['sdac']
             );
 
+            $mission_dates = array (
+                "AIA" => "2020/01/01 - Today",
+                "COSMO" => "2020/01/01 - Today",
+                "EIT" => "2020/01/01 - Today",
+                "EUI" => "2020/01/01 - Today",
+                "HMI" => "2020/01/01 - Today",
+                "LASCO" => "2020/01/01 - Today",
+                "SECCHI" => "2020/01/01 - Today",
+                "SWAP" => "2020/01/01 - Today",
+                "SXT" => "2020/01/01 - Today",
```

Comment:
Need accurate dates before pushing this.

Reply:
There are a few dates we could consider

- Mission launch date
- Official start of instrument science operations
- Earliest image helioviewer has

And on the other end
- are we still getting data from an instrument?
- end of instrument operations
- end of mission

Which ones should we include? Maybe we should have them all.

Reply:
Thinking about it now, I think it would be better to change the latest image column to show the date ranges of images we have available, then one more column that just states whether the observatory is still Active or Inactive.

Reply:
- SOHO : Inactive = MDI, Active = EIT, LASCO C2, LASCO C3
- STEREO-B : Inactive = COR1, COR2, EUVI
- STEREO-A : Active = COR1, COR2, EUVI
- PROBA2 : Active = PROBA2
- Hinode : Active = XRT
- TRACE : Inactive = TRACE
- GOES-R : Active = SUVI
- SDO : Active = AIA, HMI
- SolO : Active = EUI
- Yohkoh : Inactive = SXT
- MLSO : Active = COSMO K-Cor